proxy/config/initializers/sentry.rb

```diff
@@ -1,5 +1,5 @@
 Sentry.init do |config|
-  config.dsn = Rails.application.credentials.sentry_url
+  config.dsn = ENV["SENTRY_DSN"]
```

Comment:
This is fine but the "rails way" of doing this is to reference it like it was before and putting the ENV call in the `config/environments/` files. For a 12-factor app usually you just care about production and development there (development for doing any config to dependencies that are only included in development mode)